lib/util/problem.js

```diff
@@ -136,6 +136,9 @@ const problems = {
     // deprecated id does not exist.
     deprecatedIdNotFound: problem(404.5, ({ deprecatedId }) => `You tried to update an existing submission, but we can't find that submission to update (${deprecatedId})`),
 
+    // form version in submission does not exist
+    formVersionNotFound: problem(404.6, ({ formVersion }) => `The form version specified in this submission (${formVersion}) does not exist.`),
```

Comment:
In the case where `formVersion` is an empty string, there won't be anything between the parentheses. What do you think about putting single quotes around `formVersion`? We seem to do something similar for other Problems.